.github/workflows/_11_coverage.yml

```diff
@@ -28,9 +28,9 @@ jobs:
       - name: Setup sccache
         run: ./ci/scripts/install_sccache.sh $SCCACHE_VERSION
 
-      - name: cargo cf-test-ci --locked
+      - name: cargo cf-test-ci --locked --release
         run: |
-          cargo cf-test-ci --locked
```

Comment:
We could add the `--release` arg to the `cf-` custom commands in the `config.toml` file.

I guess it makes sense to add this for cf-test-ci and cf-clippy-ci?

Reply:
Now we can have a choice to run `cf-test-ci` or `cf-test-ci --release`, so I find it better.

If you'd insist on keeping the CI-files intact, I'd propose to also have a command `cf-test-ci-debug`. Which is a bit messy, in my opinion, but not tragically so.

Reply:
Okay. The `-ci` commands are for the CI.
CI runs in release-mode.

The `cf-test-ci` will have the `--release` baked into it.